sycl/test/.clang-format

```
@@ -1,2 +1,2 @@
 BasedOnStyle: LLVM
-CommentPragmas: "(RUN|FAIL|REQUIRES|UNSUPPORTED|CHECK) *:|expected-"
+CommentPragmas: "(RUN|FAIL|REQUIRES|UNSUPPORTED|CHECK)*:|expected-"
```

I don't think this is the right fix. 
My interpretation of this regex is "match 0 or more instances of any keywords followed by a colon" (ignoring "expected-" part).
I would expect the regex to look like:
```suggestion
CommentPragmas: "RUN|FAIL|REQUIRES|UNSUPPORTED|CHECK|expected-"
```

---

My interpretation was (is?) "match 0 or more instances of any keywords followed by a colon (anything can be put between the keyword and the colon) or match expected- (which is not followed by a colon)"